### PR TITLE
String Props Enable props

### PR DESCRIPTION
**Pull request by chrisFrazier77**
…ext prop to match

### CODE REVIEW

**Review by korgon — CHANGES_REQUESTED** [review]
Some of the props appear to not solve the issue at hand - namely that language overlays within Snap Templates will provide `lang` props that would unhide these elements that a theme may want to hide regardless of the language selection.

File: packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx
[the remark this answers is not in the document]
Comment (chrisFrazier77) [reply packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx:1]: this doesnt hide the entire button, just the text, You still render an icon in the button. So it made sense to me.

[the remark this answers is not in the document]
Comment (korgon) [reply packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx:1]: Yea, makes sense - thanks for clarifying.

**Review by korgon — APPROVED** [review]
Good to go.

**Pull request merged**